Inc/HALAL/Services/Communication/SPI/SPI.hpp

```diff
@@ -8,14 +8,17 @@
 #pragma once
 
 #include "PinModel/Pin.hpp"
-#include "Packets/SPIPacket.hpp"
+#include "Packets/SPIOrder.hpp"
 #include "DigitalOutputService/DigitalOutputService.hpp"
+#include "DigitalInputService/DigitalInputService.hpp"
 #include "DMA/DMA.hpp"
 #include "ErrorHandler/ErrorHandler.hpp"
 
 #ifdef HAL_SPI_MODULE_ENABLED
 
-#define MASTER_SPI_CHECK_DELAY 100000 //how often the master should check if the slave is ready, in nanoseconds
+#define MASTER_SPI_CHECK_DELAY 400000 //how often the master should check if the slave is ready, in nanoseconds
```

Comment:
nit pick, add NS suffix to macro, so if comment is ever lost variable holds info